src/util/jwt_utils.js

```diff
@@ -27,11 +27,35 @@ function make_internal_auth_token(object = {}, jwt_options = {}) {
     return make_auth_token(object, jwt_options);
 }
 
+/**
+ * authorize jwt token. Validate the jwt token inside passed validate_jwt method
+ * @param {string} token
+ */
 function authorize_jwt_token(token) {
+    if (token && token.includes('Bearer ')) {
+        token = token.split(' ')[1];
+    }
+    return jwt.verify(token, get_jwt_secret());
+}
+
+/**
+ * autheticate jwt token for prometheus metrics and version request
+ * @param {nb.S3Request} req
+ */
+function authenticate_jwt_token(req) {
```

Comment:
this function doesn't belong in jwt_utils. lets move it to http_utils and rename it to something like authorize_bearer(req, res) so that it handles extracting the token from the header, calling jwt_utils.authorize_jwt_token(token) and then handles the errors to return a proper http AccessDenied 403 error.